numba/tests/test_array_manipulation.py

```diff
@@ -584,6 +588,38 @@ def check(pyfunc):
     def test_transpose_array_npm(self):
         self.test_transpose_array(flags=no_pyobj_flags)
 
+    def test_swapaxes_array(self, flags=enable_pyobj_flags):
```

Comment:
It'd be good to include a case which is the no-op, i.e. axes are equal, and also some different shaped arrays. Also to check errors for out of bounds axis on both axis args.